input/resources/StructureDefinition-profile-patient.json

```diff
@@ -44,6 +44,216 @@
+        "fixedString": "Medicaid ID"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.text",
+        "path": "Patient.identifier.type.text",
+        "definition": "A human language representation of the concept as seen/selected/uttered by the user who entered the data and/or which represents the intended meaning of the user.",
+        "fixedString": "Patient Medicaid number"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "https://signalbhn.org/fhir/sid/us-medicaidid"
+      },
+      {
+        "id": "Patient.identifier:MedicareID",
+        "path": "Patient.identifier",
+        "sliceName": "MedicareID",
+        "short": "Medicare identifier for this patient",
+        "definition": "Medicare identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medicare ID"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.text",
+        "path": "Patient.identifier.type.text",
+        "fixedString": "Patient Unique Medicare Number"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "http://hl7.org/fhir/sid/us-mbi"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId",
+        "path": "Patient.identifier",
+        "sliceName": "ProviderClientId",
+        "short": "ProviderClient identifier for this patient",
+        "definition": "ProviderClient identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medical Record Number"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "PROVIDER SYSTEM URL"
+      },
+      {
+        "id": "Patient.identifier:ClientId",

```

Comment:
Since this represents the legacy Signal Beacon identifier, assign the `Patient.identifier:ClientID.system" as `https://signalbhn.org/fhir/sid/beacon-legacy`

Reply:
system value changed